### ISSUE TITLE
[Accessibility] ContextualMenu: keyboard navigation does not work when menu is not near menu button

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#7622]
### Environment Information 
 
 
- __Package version(s)__: 6.110.2 
- __Browser and OS versions__: (fill this out if relevant)

#### Actual behavior:
 
We believe there may have been a regression in fabric the broke one of the ContextualMenus in Planner. Moving the contextualMenu close to the button which opens it in the DOM seems to have fixed the issue.

Symptoms: when the contextual menu was opened, pressing tab would result in focus leaving the menu entirely. Arrow keys did not move focus to the other menu items. Pressing ESC did not close the menu.
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: Low - we have a work-around fix. 
 
Products/sites affected: Planner

**Comment by daweffe** [comment]
See the code review for the work-around fix for clues on how to reproduce:
https://office.visualstudio.com/Project/_git/Plex/pullrequest/216307?_a=overview

**Comment by daweffe** [comment]
This probably had something to do with event bubbling in React 16. There is likely nothing fabric can do to fix this. But I will leave it here as a reference in case anyone else experiences this issue.

**Comment by daweffe** [comment]
This probably had something to do with event bubbling in React 16. There is likely nothing fabric can do to fix this. But I will leave it here as a reference in case anyone else experiences this issue.
